File: Programas Dev-C++/EntradasYSalidas.cpp
/*LENGUAJE C
Entradas y Salidas
*/
//Comentario por linea

#include<stdio.h> 

int main(){ 
	char sexo;
	int edad;
	float altura;
	
	//si el tipo de dato en la funcion scanf es entero se ocupa la letra d
	//si el tipo de dato en la funcion scanf es flotante se ocupa la letra f
	//si el tipo de dato en la funcion scanf es char se ocupa la letra c
		
	printf("Digite su sexo: ");
	scanf("%c",&sexo);
	printf("Digite su edad: ");
	scanf("%d",&edad);
	printf("Digite su altura: ");
	scanf("%f",&altura);
		
	// \t ->genera una sangria
	// \n ->sirve para un salto de linea
	
	printf("\tMostrando datos del usuario\n");
	printf("\tSexo: %c\n",sexo);
	printf("\tEdad: %d\n",edad);
	printf("\tAltura: %.2f\n",altura);
	
	//.2 solo muestra dos decimales en flotante
	
	return 0;
	
}

File: Programas Python/EntradasYSalidas.py
# -*- coding: utf-8 -*-
"""
Lenguaje Python
"""

genero=input("Digite su genero: ")
edad=int(input("Digite su edad: "))
estatura=float(input("Digite su estatura: "))

print("\tMostrando datos del usuario:\n")
print(f"Edad: {edad}\n")
print(f"Genero: {genero}\n")
print(f"Estatura: {estatura}\n")

imc=estatura/3.2
print(imc)
